components/_expanding-components/uncheck-radio-button.js

```diff
@@ -0,0 +1,37 @@
+export const UncheckRadioButton = function(elementClass) {
+  const expandingComponent = elementClass
+  let expandingElement
+
+  const cacheDom = function() {
+    expandingElement = document.querySelectorAll('.' + expandingComponent)
```

Comment:
there is no reason for expandingComponent variable as you already have elementClass

Reply:
well since this code was written more for expanding components its a good name now, if we want to use this somewhere else in the future then I agree, and then we should refactor. For now its only for expanding components so the name is relevant.